.env.sample

```diff
@@ -7,7 +7,12 @@ SERVICE_ADMIN_PK=
 
 ETH_RPC_URL=
 POLYGON_RPC_URL=
+ARBITRUM_ALCHEMY_RPC_URL=
+ARBITRUM_ANKR_RPC_URL=
+
 
 # Only need the api key we are using
 ETH_ALCHEMY_API_KEY=
 ETH_ETHERSCAN_API_KEY=
+ARBITRUM_ARBISCAN_API_KEY=
```

Comment:
we're not using ARBISCAN anywhere right? think we can delete this

Reply:
sg, yeah i just had it since we have etherscan too